src/pudl/metadata/templates/codemetadata.rst.jinja

```diff
@@ -0,0 +1,33 @@
+{% if is_header %}
+=====================================================================================================
+PUDL Code Metadata
+=====================================================================================================
+The following tables map from codes used in raw data to labels used in the PUDL database.
+Also included is a table of non-standard codes that are mapped to canonical, standardized codes.
+{% endif %}
+
+.. _{{ Encoder.name }}:
```

Comment:
I could be wrong but I think [explicit targets like this](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets) need to be globally unique within the documentation, and since we're using the table name already in the PUDL DB data dictionary, there's going to be a collision here. I'm surprised Sphinx didn't complain about this during the docs build.

Ah, looks like [it threw some warnings](https://readthedocs.org/api/v2/build/15740751.txt) but not errors in the RTD build. Did you try building the docs locally with `tox -e docs`?

Do you have an account on Read The Docs? If so I can add you as a maintainer so you can see the builds (if they aren't public). I tried to add `katie-lamb` but it said that account didn't exist.

```
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:207: WARNING: duplicate label contract_types_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:267: WARNING: duplicate label energy_sources_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:315: WARNING: duplicate label entity_types_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:527: WARNING: duplicate label fuel_transportation_modes_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:554: WARNING: duplicate label fuel_types_aer_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:2233: WARNING: duplicate label prime_movers_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
/home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/pudl_db.rst:2354: WARNING: duplicate label sector_consolidated_eia, other instance in /home/docs/checkouts/readthedocs.org/user_builds/catalystcoop-pudl/checkouts/1388/docs/data_dictionaries/codes_and_labels.rst
```

Reply:
Yeah, I was experimenting with either suppressing warnings or adding a prefix but that wasn't working. I pushed it because my `tox -e docs` was having an issues before so I wasn't sure if it was just on my machine.

Reply:
Ah will read more about explicit targets and suffixes. Probably better than suppressing the warning